backup-restore/hotfixes/2.8.1/br-pre-install-patch281.sh

```diff
@@ -9,6 +9,22 @@ LOG=/tmp/br-pre-install-patch281_$$_log.txt
 exec &> >(tee -a $LOG)
 echo "Logging output in $LOG"
 
+OC_VERSION=$(oc version | awk  '/^Server Version:/ { print $NF }')
```

Comment:
@ajaylun check should be with oc get clusterversion , is not it ?

Reply:
They both provide exact same data, "oc version" most probably looking at clusterversion cr internally.